### PR TITLE
Added support for TIN geometry type

### PR DESCRIPTION
**Pull request by sebastianmattar**
... and cleaned up a bit

Note I am kind of new to this, so please be extra careful :)

### CODE REVIEW

**Comment by bertt** [comment]
looks nice! Can you add some testdata and a testcase in the pg2b3dm.database.tests project?

**Comment by sebastianmattar** [comment]
Pushed a simple test, could you have a look?

**Comment by bertt** [comment]
test fails https://github.com/Geodan/pg2b3dm/actions/runs/9775257856/job/26986267758?pr=178

**Comment by bertt** [comment]
something else goes wrong with the tests, I'll take a look

**Comment by sebastianmattar** [comment]
Seems to be dependency problem. Have the feeling this could be a version conflict between https://www.myget.org/feed/bertt/package/nuget/SharpGLTF.Toolkit/1.0.0 and https://www.nuget.org/packages/SharpGLTF.Toolkit/1.0.0

**Comment by bertt** [comment]
ah yes good suggestion, I thought everything from myget was deleted... strange it does work well on https://github.com/geodan/pg2b3dm

**Comment by bertt** [comment]
fixed the test, I've deleted the package on https://www.myget.org/feed/bertt/package/nuget/SharpGLTF.Toolkit/1.0.0

File: src/wkb2gltf.core.tests/wkb2gltf.tests.csproj [hunk 1]
```diff
@@ -19,6 +19,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageReference Include="SharpGLTF.Toolkit" Version="1.0.0" />
```
Comment (bertt) [inline src/wkb2gltf.core.tests/wkb2gltf.tests.csproj:22]: this is not needed, reference to SharpGltf is defined in project wkb2gltf.core

Comment (sebastianmattar) [reply src/wkb2gltf.core.tests/wkb2gltf.tests.csproj]: Removed it

File: src/b3dm.tileset/b3dm.tileset.csproj [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Npgsql" Version="8.0.3" />
+    <PackageReference Include="SharpGLTF.Toolkit" Version="1.0.0" />
```
Comment (bertt) [inline src/b3dm.tileset/b3dm.tileset.csproj:11]: remove

File: src/pg2b3dm.database.tests/pg2b3dm.database.tests.csproj [hunk 3]
```diff
@@ -19,6 +19,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="SharpGLTF.Toolkit" Version="1.0.0" />
```
Comment (bertt) [inline src/pg2b3dm.database.tests/pg2b3dm.database.tests.csproj:22]: remove

Comment (sebastianmattar) [reply src/pg2b3dm.database.tests/pg2b3dm.database.tests.csproj]: Done

File: src/b3dm.tileset/b3dm.tileset.csproj (hunk 2 above)
Comment (sebastianmattar) [reply src/b3dm.tileset/b3dm.tileset.csproj]: Done

File: src/b3dm.tileset.tests/b3dm.tileset.tests.csproj [hunk 4]
```diff
@@ -18,6 +18,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageReference Include="SharpGLTF.Toolkit" Version="1.0.0" />
```
[the remark this answers is not in the document]
Comment (sebastianmattar) [reply src/b3dm.tileset.tests/b3dm.tileset.tests.csproj]: Done

**Comment by bertt** [comment]
Thanks, nice addition! Will be in next release.

**Pull request merged**